wis2box-create-config.py

```diff
@@ -344,6 +344,11 @@ def create_wis2box_env(config_dir: str) -> None:
         # update WIS2BOX_BROKER_PUBLIC settings after updating broker defaults
         fh.write('# update WIS2BOX_BROKER_PUBLIC settings after updating broker defaults\n') # noqa
         fh.write('WIS2BOX_BROKER_PUBLIC=mqtt://${WIS2BOX_BROKER_USERNAME}:${WIS2BOX_BROKER_PASSWORD}@mosquitto:1883\n') # noqa
+        # use the default username wis2box for WIS2BOX_CAP
```

Comment:
Not all users will use wis2box to publish CAP messages, could we make this environment variables optional ?

Reply:
Agree, optional.